tests/livestore/init_test.go

```diff
@@ -16,28 +16,27 @@ import (
 // hostname can be dynamic, dependent on whether we are running on CI or locally.
 var hostname = "localhost:5432"
 
-// TestMain runs before each test run.
```

Comment:
Is this github doing something funcky or did you remove these comments? :D

Reply:
Nope! My bad. I tidied this up so we could use `defer` and accidentally removed the comment.